zebra/rt_netlink.c

```diff
@@ -1070,6 +1077,21 @@ static int netlink_route_change_read_multicast(struct nlmsghdr *h,
 		}
 	}
 
+	if (rtm->rtm_family == RTNL_FAMILY_IPMR) {
+		m->src.family = AF_INET;
+		m->grp.family = AF_INET;
+		memcpy(&m->src.u.prefix4, src, sizeof(m->src.u.prefix4));
+		memcpy(&m->grp.u.prefix4, grp, sizeof(m->grp.u.prefix4));
+	} else if (rtm->rtm_family == RTNL_FAMILY_IP6MR) {
+		m->src.family = AF_INET6;
+		m->grp.family = AF_INET6;
+		memcpy(&m->src.u.prefix6, src, sizeof(m->src.u.prefix6));
+		memcpy(&m->grp.u.prefix6, grp, sizeof(m->grp.u.prefix6));
+	} else {
+		zlog_warn("%s: Invalid rtm_famlily received", __func__);
```

Comment:
typo rtm_famlily

Reply:
I probably need glasses 😄 . Thanks for pointing it out. Have fixed it.